neo4j/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
⭐ For not committing sensitive secrets (e.g. API keys) unencrypted to the repository.